MovieQuiz/Presentation/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  MovieQuizPresenter.swift
+//  MovieQuiz
+//
+//  Created by Илья Валито on 28.10.2022.
+//
+
+import UIKit
+
+final class MovieQuizPresenter {
+
+    private weak var viewController: MovieQuizViewControllerProtocol?
+    private var questionFactory: QuestionFactoryProtocol?
+    private var statisticService: StatisticService!
+    private var currentQuestion: QuizQuestion?
+    private let questionsAmount: Int = 10
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+    
+    init(viewController: MovieQuizViewControllerProtocol) {
+            self.viewController = viewController
+            
+            statisticService = StatisticServiceImplementation()
+            questionFactory = QuestionFactory(moviesLoader: MoviesLoader(), delegate: self)
+            questionFactory?.loadData()
+            viewController.showLoadingIndicator()
+        }
```

Comment:
Не забывайте исправлять форматирование (control +i)
```suggestion
        self.viewController = viewController
            
        statisticService = StatisticServiceImplementation()
        questionFactory = QuestionFactory(moviesLoader: MoviesLoader(), delegate: self)
        questionFactory?.loadData()
        viewController.showLoadingIndicator()
    }
```